src/components/editor/Homepage.jsx

```diff
@@ -0,0 +1,188 @@
+    <p>Infobar description</p>
+    <input placeholder="Infobar description" defaultValue={description} value={description} id={`section-${sectionIndex}-infobar-description`} onChange={onFieldChange} />
+    <p>Infobar button name</p>
+    <input placeholder="Infobar button name" defaultValue={button} value={button} id={`section-${sectionIndex}-infobar-button`} onChange={onFieldChange} />
+    <p>Infobar button URL</p>
+    <input placeholder="Infobar button URL" defaultValue={url} value={url} id={`section-${sectionIndex}-infobar-url`} onChange={onFieldChange} />
+    <br />
+    <button type="button" id={`section-${sectionIndex}`} onClick={deleteSection}>Delete section</button>
+  </div>
+);
+
+const EditorResourcesSection = ({
+  title, subtitle, button, sectionIndex, deleteSection, onFieldChange,
+}) => (
+  <div className={styles.card}>
+    <p><b>Resources section</b></p>
+    <p>Resources section title</p>
+    <input placeholder="Resource section title" defaultValue={title} value={title} id={`section-${sectionIndex}-resources-title`} onChange={onFieldChange} />
+    <p>Resources section subtitle</p>
+    <input placeholder="Resource section subtitle" defaultValue={subtitle} value={subtitle} id={`section-${sectionIndex}-resources-subtitle`} onChange={onFieldChange} />
+    <p>Resources button name</p>
+    <input placeholder="Resource button button" defaultValue={button} value={button} id={`section-${sectionIndex}-resources-button`} onChange={onFieldChange} />
+    <br />
+    <button type="button" id={`section-${sectionIndex}`} onClick={deleteSection}>Delete section</button>
+  </div>
+);
+
+const KeyHighlight = ({
+  title,
+  description,
+  url,
+  highlightIndex,
+  deleteHighlight,
+  createHighlight,
+  onFieldChange,
+  allowMoreHighlights,
+}) => (
+  <div className={styles.card} key={highlightIndex}>
+    <b>
+Highlight
+      {highlightIndex}
+    </b>
+    <p>Highlight title</p>
+    <input placeholder="Highlight title" defaultValue={title} value={title} id={`highlight-${highlightIndex}-title`} onChange={onFieldChange} key={`${highlightIndex}-title`} />
+    <p>Highlight description</p>
+    <input placeholder="Highlight description" defaultValue={description} value={description} id={`highlight-${highlightIndex}-description`} onChange={onFieldChange} key={`${highlightIndex}-description`} />
+    <p>Highlight URL</p>
+    <input placeholder="Highlight URL" defaultValue={url} value={url} id={`highlight-${highlightIndex}-url`} onChange={onFieldChange} key={`${highlightIndex}-url`} />
+    <br />
+    <button type="button" id={`highlight-${highlightIndex}-delete`} onClick={deleteHighlight} key={`${highlightIndex}-delete`}>Delete highlight</button>
+    {allowMoreHighlights
+      ? <button type="button" id={`highlight-${highlightIndex}-create`} onClick={createHighlight} key={`${highlightIndex}-create`}>Create highlight</button>
+      : null}
+  </div>
```

Comment:
If all key highlights are deleted there is no way to add them back.
Would it be an issue that needs to be fixed now or do we just create an issue to track it?

Reply:
Creating an issue to track it - will fix it in the next PR from the `homepage-continued` branch.